src/components/settings/settings-access-status-types/settings-access-status-types.js

```diff
@@ -0,0 +1,163 @@
+
+import { IntlConsumer } from '@folio/stripes/core';
+import {
+  Pane,
+  PaneCloseLink,
+  NoValue,
+  TextField,
+} from '@folio/stripes/components';
+import { EditableList } from '@folio/stripes/smart-components';
+
+const SettingsAccessStatusTypes = ({
+  accessTypesData,
+  onCreate,
+  onDelete,
+  onUpdate,
+}) => {
+  const onCreateItem = ({ attributes }) => onCreate({
+    type: 'accessTypes',
+    attributes,
+  });
+
+  const formatter = {
+    name: ({ attributes }) => (attributes?.name ?? <NoValue />),
+    description: ({ attributes }) => (attributes?.description ?? <NoValue />),
+    lastUpdated: (data) => {
+      const user = data.updater ?? data.creator;
+
+      return data.metadata ? (
+        <FormattedMessage
+          id="ui-eholdings.settings.accessStatusTypes.lastUpdated.data"
+          values={{
+            date: <FormattedDate value={data.metadata.updatedDate} />,
+            name: `${user?.lastName} ${user?.firstName}`,
+          }}
+        />
+      ) : <NoValue />;
+    },
+    // records will be done after MODKBEKBJ-378
+    records: () => <NoValue />
+  };
+
+  // eslint-disable-next-line react/prop-types
+  const renderField = ({ fieldIndex, rowIndex, name }, validate) => {
+    const dataAttr = { [`data-test-settings-access-type-${name}-field`]: true };
+
+    return (
+      <div
+        {...dataAttr}
+        key={fieldIndex}
+      >
+        <Field
+          component={TextField}
+          name={`items[${rowIndex}].attributes.${name}`}
+          validate={validate}
+          marginBottom0
+          fullWidth
+          autoFocus={name === 'name'}
+        />
+      </div>
+    );
+  };
+
+  const nameValidation = value => {
+    if (!value) {
+      return <FormattedMessage id="ui-eholdings.settings.accessStatusTypes.type.validation" />;
+    } else if (value && value.length > 75) {
+      return <FormattedMessage
+        id="ui-eholdings.settings.accessStatusTypes.validation"
+        values={{ limit: 75 }}
+      />;
+    } else {
+      return null;
+    }
+  };
+
+  const descriptionValidation = value => {
+    if (value && value.length > 150) {
+      return <FormattedMessage
+        id="ui-eholdings.settings.accessStatusTypes.validation"
+        values={{ limit: 150 }}
+      />;
+    } else {
+      return null;
+    }
+  };
+
+  return (
+    <Pane
+      paneTitle={<FormattedMessage id="ui-eholdings.settings.accessStatusTypes" />}
+      data-test-settings-access-status-types
+      defaultWidth="fill"
+      firstMenu={(
+        <FormattedMessage id="ui-eholdings.settings.goBackToEholdings">
+          {ariaLabel => (
+            <PaneCloseLink
+              ariaLabel={ariaLabel}
+              to="/settings/eholdings"
+            />
+          )}
+        </FormattedMessage>
+      )}
+    >
+      <IntlConsumer>
+        {intl => (
+          <EditableList
+            actionProps={accessTypesData && accessTypesData.length >= 15 ? { create: () => ({ disabled: true }) } : {}}
```

Comment:
I would recommend you to bring current expression into separate function `getActionProps` or smth like that.
And also would be better to use `isEmpty()` lodash method instead `accessTypesData.length`